app/src/pages/home.jsx

```diff
@@ -0,0 +1,53 @@
+import React, { useState } from 'react'
+import { Link } from 'react-router-dom'
+import Logo from '../images/logolu.png'
+import Intereses from '../components/intereses'
```

Comment:
The order to import things should be external or aliased libraries, then absolute paths (`../../..`) and then the others